pkgs/applications/audio/mousai/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, python3
+, wrapGAppsHook
+, gobject-introspection
+, gtk3
+, desktop-file-utils
+, libhandy
+, gst_all_1
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "mousai";
+  version = "0.2.0";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "SeaDve";
+    repo = "Mousai";
+    rev = "v${version}";
+    sha256 = "10g3wj9yy15yzyfa1l96i97i3k6blm064p49hn7hp001cmc3w045";
+  };
+
+  postPatch = ''
+    chmod +x build-aux/meson/postinstall.py
+    patchShebangs build-aux/meson/postinstall.py
+  '';
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook
+    gobject-introspection
+    python3
+    desktop-file-utils
+    python3.pkgs.wrapPython
+  ];
+
+  buildInputs = [
+    gtk3
+    gobject-introspection
+    libhandy
+  ] ++ (with gst_all_1; [
+    gstreamer
+    gst-plugins-good
+    gst-plugins-ugly
+    gst-plugins-base
+  ]);
+
+  propagatedBuildInputs = with python3.pkgs; [
+    pygobject3
+    gst-python
+    requests
+  ];
+
+  preFixup = ''
+    wrapPythonPrograms
+  '';
```

Comment:
We need to do something about this tripple wrapping.

```ShellSession
$ results/mousai/bin/
mousai                             .mousai-wrapped_                   ...mousai-wrapped-wrapped-wrapped
.mousai-wrapped                    ..mousai-wrapped-wrapped
```

Reply:
I managed to have it only double-wrapped
```
ls /nix/store/pgf869n0wjb2k9s31p1q2858wwkncijl-mousai-0.2.0/bin -a
.  ..  mousai  .mousai-wrapped  ..mousai-wrapped-wrapped
```
Took the concept from here https://github.com/NixOS/nixpkgs/blob/a230fd4368bf8d3db02800b33cbb550d21a5a1aa/pkgs/applications/misc/openlp/default.nix